app/lib/queryBuilder.js

```diff
@@ -15,21 +15,24 @@ function getOpenPharmacyFilter(date) {
   // the multi-line format below is useful for legibility but not
   // needed by azure search and so is stripped out by the whitespace
   // replace.
+  // Note: Additional opening times can have a type of _either_ Additional or General.
+  // This is an error in the source data which seems to have been introduced in 2013
+  // and never corrected.
+  // I think that additional hours should be of type general to allow, for example,
+  // a GP surgery to have different hours on a bank holiday for reception only.
   return `
   ${pharmacyFilter} and
   ( OpeningTimesV2/any(time:
-          time/IsOpen
           and time/Weekday eq '${weekday}'
           and time/OpeningTimeType eq 'General'
```

Comment:
I presume this doesn't need updating to include a `Additional OpeningTimeType` clause because the data is such that these times are only ever `General`?

Reply:
That's correct.